### ISSUE TITLE
Sort bug when using cookie extension

### ISSUE THREAD

**Issue by olafcm** [wenzhixin/bootstrap-table#5952]
### Bootstraptable version(s) affected

1.19.1

### Description

Sort setting is ignored when cookie extension is active for table

### Example(s)

https://live.bootstrap-table.com/code/olafcm/9630 (cookie enabled, sort setting ignored)
https://live.bootstrap-table.com/code/olafcm/9631 (no cookie enabled, sort setting correctly applied)

**Comment by rvaandrager** [comment]
The following code snippet in bootstrap-table-cookie.js (around line 2522) causes this bug:

```
this.options.sortOrder = undefined; // sortName

this.options.sortName = undefined;

if (!sortPriorityCookie) {
  // sortOrder
  this.options.sortOrder = sortOrderCookie ? sortOrderCookie : this.options.sortOrder; // sortName

  this.options.sortName = sortOrderNameCookie ? sortOrderNameCookie : this.options.sortName;
} // sortPriority
```

Here, the values for `options.sortOrder` and `options.sortName` are set to null first, and then they are copied again in the next if statement. This should be the other way around: when not using `sortPriorityCookie`, set `sortOrderCookie` to `options.sortOrder`, otherwise set it to `undefined`.

Replacing this snippet with the following resolves the problem:

```
if (!sortPriorityCookie) {
  // sortOrder
  this.options.sortOrder = sortOrderCookie ? sortOrderCookie : this.options.sortOrder; // sortName

  this.options.sortName = sortOrderNameCookie ? sortOrderNameCookie : this.options.sortName;
} // sortPriority
else {
  this.options.sortOrder = undefined; // sortName

  this.options.sortName = undefined;
}
```

**Comment by balajipulluru** [comment]
Could you please update when this fix will be released.
Thanks.

**Comment by balajipulluru** [comment]
I see the issue is fixed and the code merged in #5978
Could you please update when this fix will be released
Thanks for the fix guys.
